setup.cfg

```diff
@@ -1,2 +1,44 @@
+[metadata]
+name = astor
+description = Read/rewrite/write Python ASTs
+long_description = file:README.rst
+author = Patrick Maupin
+email = pmaupin@gmail.com
+platforms = Independent
+url = https://github.com/berkerpeksag/astor
+license = BSD-3-Clause
+keywords = ast, codegen, PEP8
```

Comment:
I always wanted to fix this so let's rename PEP8 to PEP 8 :)

Reply:
OK